api/models/release.go

```diff
@@ -171,6 +171,7 @@ func (r *Release) Promote() error {
 	}
 
 	app.Parameters["SubnetsPrivate"] = subnetsPrivate
+	app.Parameters["VPCCIDR"] = os.Getenv("VPCCIDR")
```

Comment:
Change looks good! Great catch.

Minor style feedback, I'd put this on line 161. That's the set of default static parameters. Notice also those keys are in alphabetical order.